pkgcore/operations/format.py

```diff
@@ -57,7 +58,9 @@ def fetch_one(self, fetchable, observer):
     def failed_fetch(self, fetchable, observer):
         # run pkg_nofetch phase for fetch restricted pkgs
         if 'fetch' in self.pkg.restrict:
-            build_ops = self.domain.build_pkg(self.pkg, observer)
```

Comment:
Oh, a little misunderstanding =) Anyway, I guess it's ready for merge and you might close those two bugs.